src/Provider/Authentication.php

```diff
@@ -119,4 +121,15 @@ protected function get_request_path(): string {
 
 		return '/' . ltrim( $request_path, '/' );
 	}
+
+	public function user_has_cap( array $allcaps, array $caps, array $args ) {
```

Comment:
The method has now been renamed to `maybe_allow_public_access`.
I also dropped the `$caps` and `$args` parameters as well as adjusted the number of parameters in the `add_filter()` call.

Reply:
You can leave off the optional parameters in the `add_filter()` call since `10` and `1` are the defaults. Once that's updated, I'll go ahead and get this merged. Everything else looks good. Thanks!